src/fiskaltrust.AndroidLauncher.Common/fiskaltrust.AndroidLauncher.Common.csproj

```diff
@@ -105,6 +106,12 @@
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="fiskaltrust.Middleware.Interface.Client.Grpc">

```

Comment:
We shouldn't add that here - otherwise, both APKs will contain both clients (and the HTTP package will be as big as the gRPC one)